lib/tasks/storage.rake

```diff
@@ -1,4 +1,18 @@
 namespace :google_drive do
+  desc "Create supporting_document events for past document uploads"
+  task events_backfill: [:environment] do
+    Document.find_each do |document|
+      Event.new.trigger(
+        :supporting_document_created,
+        vacancy_id: StringAnonymiser.new(document.vacancy_id),
+        name: document.name,
+        size: document.size,
+        content_type: document.content_type,
+        created_at: document.created_at,
```

Comment:
it should be `document.created_at.to_s(:iso8601)`

Reply:
This goes through this bit of code: https://github.com/DFE-Digital/teaching-vacancies/blob/master/app/services/event.rb#L64

So it will already be converted into ISO8601 (and then go into a date column in BQ)